### ISSUE TITLE
Module not found Error Occurs in nextJs14.0.3

### ISSUE THREAD

**Issue by humuhimi** [OvidijusParsiunas/deep-chat#64]
Hi @OvidijusParsiunas thanks your deep-chat.

Description:
I'm using deep-chat on my website and I encounter Module not found Error when I bump next from 14.0.2 to 14.0.3.
plz tell me how to fix it.

```tsx
import dynamic from 'next/dynamic';

interface BotButtonProps extends React.HTMLProps<HTMLElement> {
  t: TFunction;
}
const BotButton = ({ t }: BotButtonProps) => {
  const DeepChat = dynamic(
    () => import('deep-chat-react').then((mod) => mod.DeepChat),
    {
      ssr: false,
    },
  );
```

```error
./src/components/BotButton.tsx
Module not found: ESM packages (deep-chat-react) need to be imported. Use 'import' to reference the package instead. https://nextjs.org/docs/messages/import-esm-externals

https://nextjs.org/docs/messages/module-not-found
```

Environment:
- Browser: Chrome version 119.0.6045.200
- nextjs version: 14.0.3
- Node.js version: v19.1.0

**Comment by OvidijusParsiunas** [comment]
Hi @humuhimi.
I tracked the recent `issues` in the Next repo and it appears that this is a bug that was introduced in the `14.0.3` release. Fortunately they have merged a fix for this yesterday in the following [PR](https://github.com/vercel/next.js/pull/59246) and it is now in the `Canary` branch which will be released in `14.0.4`. I am personally not sure when this version will exactly be released, but it should be soon.

You can probably use the `14.0.4-canary.43` version instead - but keep note that this is experimental. You can check [here](https://www.npmjs.com/package/next?activeTab=versions) for all other available versions.

Because this is an issue on the NextJs side it should be tracked in their resources and I will be closing this issue.
Thankyou for letting me know about this @humuhimi!

**Comment by humuhimi** [comment]
thank you! I will waiting for it!
